src/bin/generate.ts

```diff
@@ -19,6 +20,15 @@ const writeFile = promisify(_writeFile);
 const mkdirp = promisify(_mkdirp);
 const ncp = promisify(_ncp);
 
+async function group<T>(name: string, callback: () => Promise<T>): Promise<T> {
+  if (process.env.CI) {
+    return github.group(name, callback);
+  } else {
+    console.log(`\n${name}`);
+    return callback();
+  }
+}
+
```

Comment:
should we document this? Just thinking that if someone ends up needing to do work around github actions, it might not be clear what this is doing/what it's for